project_report/tex/evaluation.tex

```diff
@@ -1 +1,26 @@
 \section{Evaluation}
+The evaluation section describes how the reflections of the goals, set for the
+project, meets the customer demands, as well as how the development process has
+been.
+
+
+\subsection{Evaluation from the customer side}
+The group has managed to meet the requirements, set in the initial requirements
+analysis, in the beginning of the project. The created system is able to control
+the production line, using the dashboard, and is able to collect, display and
+store relevant data in a database. The data is used for further analysis, such
+as calculating the OEE and optimal machine speed, as well as estimating the
+error function.
+
+
+\subsection{Development process}
```

Comment:
You are stating some facts, but you never evaluate them. 
Consider writing about:
- How did UP and Scrum work for us?
  - Along with all the artefacts from UP (did they help us develop the system)
- How well did we answer the requirements?

Reply:
Maybe we have done it now?